spec/integration/date_time_properties_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
+    class Foo < ActiveFedora::Base
+      property :date, predicate: ::RDF::DC.date do |index|
+        index.type :date
```

Comment:
Is the "type" necessary?  It should be inferred from what is given to the indexer, right?